integration-libs/cdc/core/store/effects/cdc-user-addresses.effect.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { CdcJsService } from '@spartacus/cdc/root';
+import {
+  Address,
+  Country,
+  GlobalMessageService,
+  GlobalMessageType,
+  UserActions,
+  UserAddressConnector,
+  UserAddressService,
+  UserIdService,
+} from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { mergeMap, switchMap, take, tap } from 'rxjs/operators';
+
+@Injectable()
+export class CdcUserAddressesEffects {
+  addressFieldKeys = ['line1', 'line2', 'region.name', 'town', 'postalCode'];
+
+  cdcAddUserAddress$ = createEffect(
+    () =>
+      this.actions$.pipe(
+        ofType(UserActions.ADD_USER_ADDRESS_SUCCESS),
+        mergeMap(() => {
+          return this.updateDefaultAddressInCDC().pipe(
+            tap((error) => this.showErrorMessage(error))
```

Comment:
```suggestion
            tap({ error: (error) => this.showErrorMessage(error)} )
```
We don't want to run it on successful emissions, but only on rxjs error emissions.

The same applies to a few other effects in this file

Reply:
noted, thanks for the catch @Platonn